CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Formatted as described on [https://keepachangelog.com](https://keepachangelog.co
 
 ## [Unreleased]
 
+
+### [2.3.1] (2024-09-17)
```

Comment:
Also add link at bottom of page

Reply:
It's not in the CONTRIBUTING.md.

I'll add that